components/Multimedia/UserAvatar.vue

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<template>
+    <div
+        class="user-avatar"
+        :style="avatarStyles">
+        <Picture
+            v-if="imageId"
+            fab
+            :image-id="imageId" />
+        <span
+            v-else
+            class="user-avatar__placeholder font--medium-14-20"
+            v-text="avatarInitial" />
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'UserAvatar',
+    components: {
+        Picture: () => import('~/core/components/Multimedia/Picture'),
+    },
+    props: {
+        imageId: {
+            type: String,
+            default: '',
+        },
+        userName: {
+            type: String,
+            default: '',
+        },
+        pictureSize: {
+            type: Number,
+            default: 24,
+        },
+    },
+    computed: {
+        avatarInitial() {
+            return this.userName.length ? this.userName[0] : '';
+        },
+        avatarStyles() {
```

Comment:
What about breaking convention with Classes, Styles at the end of the name if the return value is contextual - like here, you are setting an image bounds/size. Maybe lets simplify it to avatarSize?